model/transaction/event.go

```diff
@@ -170,6 +170,13 @@ func (e *Event) Transform(tctx *transform.Context) []beat.Event {
 	utility.Add(fields, "user_agent", e.User.UserAgentFields())
 	tctx.Metadata.Merge(fields)
 
+	utility.Add(fields, "http", m.HttpFields(e.Context))
+	utility.Add(fields, "url", m.UrlFields(e.Context))
+	delete(e.Context, "request")
+	delete(e.Context, "response")
+
+	utility.Add(fields, "context", e.Context)
+
 	events = append(events, beat.Event{Fields: fields, Timestamp: e.Timestamp})
 
 	return events
```

Comment:
What happens with the `context` in `span`?

Reply:
https://github.com/elastic/apm-server/issues/1804 open to discuss